<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterchoiseDialog</class>
 <widget class="QDialog" name="FilterchoiseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose filter type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="borderFilterRadioButton">
     <property name="text">
      <string>Border Filter</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="windowFilterRadioButton">
     <property name="text">
      <string>Mean window filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="medianWindowFilterRadioButton">
     <property name="text">
      <string>Median window filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clusterFilterRadioButton">
     <property name="text">
      <string>Error cutoff filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="okCancelButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
